app/views/api/v0/follows/followers.json.jbuilder

```diff
@@ -0,0 +1,8 @@
+json.array! @follows do |follow|
+  json.type_of                 "follower"
+  json.id                      follow.follower.id
+  json.name                    follow.follower.name
+  json.path                    follow.follower.path
+  json.username                follow.follower.username
+  json.profile_image           ProfileImage.new(follow.follower).get(60)
```

Comment:
@rhymes, I noticed while writing the partial that on the dashboard organization shows the name twice:

![org](https://user-images.githubusercontent.com/9089551/67128847-4c2e3180-f1d3-11e9-8880-03c3cb67eddd.png)

The username is ``azure``, so I was expecting it to show ``@azure`` but got ``@Microsoft Azure`` instead. Shouldn't it be ``@organization.username`` instead of ``@organization.name``? Or is there a reason why the name is used?

Reply:
looks like a bug to me :)